doc/source/release-notes/Percona-Server-5.7.35-38.rst

```diff
@@ -0,0 +1,38 @@
+.. _PS-5.7.35-38:
+
+================================================================================
+*Percona Server for MySQL* 5.7.35-38
+================================================================================
+
+:Date: August 18, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.35-38
+includes all the features and bug fixes available in
+`MySQL 5.7.35 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-4641`: The `innodb_flush_method <https://www.percona.com/doc/percona-server/5.7/scalability/innodb_io.html#innodb_flush_method>`__ documentation needs more explanation.
+* :jirabug:`PS-7715`: Missing documentation on several `Data-At-Rest <https://www.percona.com/doc/percona-server/5.7/security/data-at-rest-encryption.html>`__ variables
```

Comment:
Describe / documeny missing variable for Data-at-rest encryption

Reply:
Removed.